test/models/trait_test.dart

```diff
@@ -87,4 +86,57 @@ void main() {
       expect(updated.value, 'value');
     });
   });
+
+  group('toJson', () {
```

Comment:
What about `fromJson` ? Maybe those are covered above?

Reply:
We should probably add a case for verifying that strings still work too?

Reply:
Done